release/gff/gff.byn.gff_blin/source/gff.byn.gff_blin.model.kps

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>13.0.101.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">GFF Blin Lexical Model</Name>
+    <Copyright URL="">© 2023 Ge'ez Frontier Foundation</Copyright>
+    <Author URL="mailto:yacob@geez.org">Ge'ez Frontier Foundation</Author>
+    <Version URL="">1.0</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>..\build\gff.byn.gff_blin.model.js</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.js</FileType>
+    </File>
+    <File>
+      <Name>welcome.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>readme.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+  </Files>
+  <Keyboards/>
+  <LexicalModels>
+    <LexicalModel>
+      <Name>GFF Blin</Name>
+      <ID>gff.byn.gff_blin</ID>
+      <Languages>
+        <Language ID="byn-Ethi">Blin</Language>

```

Comment:
Was this populated by Keyman Developer? We may want to add the script info like
```suggestion
        <Language ID="byn-Ethi">Blin (Ethiopic)</Language>
```


As @DavidLRowe pointed out, the .model_info should also have `byn-Ethi`.